x-pack/plugins/code/public/components/main/main.scss

```diff
@@ -259,3 +260,8 @@
     border-top: none;
   }
 }
+
+
+.codeIcon__language path {
```

Comment:
When we've don't this kind of thing before we just added a selector onto the path in the svg. I don't know if you always use the same file, but it's a little more specific than targeting a naked selector.